src/Credentials.ts

```diff
@@ -487,8 +495,8 @@ class Credentials {
    * @param    {String}            credential.exp         time at which this claim expires and is no longer valid (seconds since epoch)
    * @return   {Promise<Object, Error>}                   a promise which resolves with a credential (JWT) or rejects with an error
    */
-  createVerification({ sub, claim, exp, vc, callbackUrl }: VerificationParam) {
-    return this.signJWT({ sub, claim, exp, vc, callbackUrl })
+  createVerification({ aud, sub, jti, nbf, vc, exp, proof }: VerificationParam) {
```

Comment:
I believe it would be a better strategy to add a new method called `createCredential` to do this work and mark this one as deprecated.

Reply:
I agree, will revert the changes to `createVerification` and add a new method `createVerifiableCredential` to make this more clear and consistent with the changes being done for Presentations (https://github.com/uport-project/uport-credentials/pull/182)